quesma/elasticsearch/index.go

```diff
@@ -15,5 +15,5 @@ func IsIndexPattern(index string) bool {
 }
 
 func IsInternalIndex(index string) bool {
```

Comment:
Just for precision/clarity, I'd add one more func `IsInternalElasticsearchPath` -> `_nodes` and `_xpack` are internal endpoints, not indices